index.html

```diff
@@ -1316,9 +1322,9 @@ <h4>Verify Derived Proof (bbs-2023)</h4>
           <ol class="algorithm">
             <!-- [bbsProof, labelMapCompressed, mandatoryIndexes, adjSelectedIndexes] -->
             <li>
-Initialize `bbsProof`, `proofHash`, `mandatoryHash`, `selectedIndexes`, and
-`nonMandatory` to the values associated with their property
-names in the object returned when calling the algorithm in Section
+Initialize `bbsProof`, `proofHash`, `mandatoryHash`, `selectedIndexes`,
+`presentationHeader` and `nonMandatory` to the values associated with their
```

Comment:
I wonder if we need a note that says that implementations can surface the `presentationHeader` for external use if desired (perhaps to compare a challenge or something like that in a protocol that uses the VCDM). Nothing in the spec text prevents them from doing so, but it might be helpful to have a note like this:

> Note: Implementations are free to expose any component parsed from the proof data for external use or to compare them internally against externally provided values based on protocol needs. For example, an implementation might expose `presentationHeader` for comparison against an expected pseudo-random challenge value used in a presentation protocol.

Reply:
@dlongley I'm thinking we put this as an *informational* note at the end of section [3.3.7 Verify Derived Proof (bbs-2023)](https://w3c.github.io/vc-di-bbs/#verify-derived-proof-bbs-2023). Other ideas?

Reply:
@Wind4Greg, that sounds good.

Reply:
Normative, multiple reviews, changes requested and made, no objections, merging.